TrailBlaze/Networking/Interceptor.m

```diff
@@ -33,6 +33,11 @@ + (NSArray *) getRemainingRoutePoints: (NSArray *) routePoints runnerLocation: (
     return [routePoints subarrayWithRange:NSMakeRange(closestIndex, routePoints.count - 1)];
 }
 
-
++ (NSArray *) sortInAscendingOrder: (NSArray *) etaDifferences {
+    NSSortDescriptor *sortDescriptor;
+    sortDescriptor = [[NSSortDescriptor alloc] initWithKey:@"doubleValue" ascending:YES];
```

Comment:
Not sure why we need the variable definition and set on different lines, let's unify this:
`NSSortDescriptor *const sortDescriptor = [[NSSortDescriptor alloc] initWithKey:@"doubleValue" ascending:YES];`